core/testing/src/main/java/io/github/droidkaigi/confsched2023/testing/robot/AboutScreenRobot.kt

```diff
@@ -55,4 +60,15 @@ class AboutScreenRobot @Inject constructor(
         composeTestRule.waitForIdle()
         testDispatcher.scheduler.advanceUntilIdle()
     }
+
+    fun scrollAboutScreen() {
```

Comment:
You can provide the scrolled screenshot test to check the scrolled screen 😉  You don't have to do it in this PR though.